R/update.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
ah nice - but please move this to another (internal repo) folder, e.g. `devel` could be suitable